src/rust/ide/lib/view/src/graph_editor/component/edge.rs

```diff
@@ -573,6 +1335,16 @@ impl EdgeModelData {
         let downward_far   = -target.y > min_len_for_non_curved_line || target_is_below_node;
         let is_down        = if is_flat_side {downward_flat} else {downward_far};
 
+        // === Layout State ===
+        // Initial guess at our layout. Will be refined for some edge cases when we have more
+        // layout informaiton.
```

Comment:
What does it mean "when we have more layout information"? I mean, this should be explained better here - what is to be refined and what info is missing at this point? Just add few words about it - with an example info that is missing here that will be known later